README.md

```diff
@@ -102,6 +102,9 @@ const fft = new FFT(size);
 
 NOTE: `size` MUST be a power of two and MUST be bigger than 1.
 
+If you are looking to find the nearest power of 2 given the size of your dataset, here is a good tutorial:
```

Comment:
Nit. Could be `[good tutorial](https://stackoverflow.com/questions/466204/rounding-up-to-next-power-of-2/466256#466256)`.